lib/SIL/IR/TypeLowering.cpp

```diff
@@ -638,6 +642,12 @@ namespace {
           type, getReferenceRecursiveProperties(isSensitive));
     }
 
+    RetTy visitMoveOnlyType(CanMoveOnlyType type, AbstractionPattern origType,
+                            IsTypeExpansionSensitive_t isSensitive) {
+      return asImpl().handleReference(
```

Comment:
@jckarter @atrick my memory is that handle reference is for all loadable types. Originally I was going to add a sub-object here so that even if the initial type was trivial, I could mixin a nontrivial bit. But I found that type lowering got stuck into an infinite spiral. So for now I just left it as handleReference.